template.yml

```diff
@@ -262,6 +262,82 @@ Resources:
       Environment:
         Variables:
           AWS_LAMBDA_EXEC_WRAPPER: /opt/change_runtime_directory
+          OUTPUT_BUCKET_NAME: !Ref OutputBucket
+          GET_OUTPUT_URL: !Sub "https://${OutputDomain}"
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub  "cdo-dev-${SubDomainName}-output"
+      CorsConfiguration:
+        CorsRules:
+          - AllowedMethods: [GET]
+            AllowedOrigins: ['*']
+            AllowedHeaders: ['*']
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: 'AES256'
+      LifecycleConfiguration:
+        Rules:
+          - Id: ExpirationRule
+            Status: Enabled
+            ExpirationInDays: 1
+  OutputBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref OutputBucket
+      PolicyDocument:
+        Statement:
+        - Action: ['s3:GetObject']
+          Effect: Allow
+          Resource: !Sub "arn:aws:s3:::${OutputBucket}/*"
+          Principal: '*'
+  OutputAPICertificate:
+    Type: AWS::CertificateManager::Certificate
+    Properties:
+      DomainName: !Sub "${SubDomainName}-output.${BaseDomainName}"
+      ValidationMethod: DNS
+      DomainValidationOptions:
+        - DomainName: !Sub "${SubDomainName}-output.${BaseDomainName}"
+          HostedZoneId: !Ref BaseDomainNameHostedZonedID
+  OutputDomain:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneName: !Sub "${BaseDomainName}."
+      Name: !Sub "${SubDomainName}-output.${BaseDomainName}"
+      Type: A
+      AliasTarget:
+        DNSName: !GetAtt OutputCDN.DomainName
+        HostedZoneId: Z2FDTNDATAQYW2 # static ID for cloudfront aliases
+  OutputCDN:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Enabled: true
+        Aliases: [!Sub "${SubDomainName}-output.${BaseDomainName}"]
+        ViewerCertificate:
+          AcmCertificateArn: !Ref OutputAPICertificate
+          MinimumProtocolVersion: TLSv1
+          SslSupportMethod: sni-only
+        CustomErrorResponses:
+          - ErrorCode: 403
+            ErrorCachingMinTTL: 0
+        # TODO: enable logging when LogBucket is set up
+        # Logging:
```

Comment:
Seems like we don't have a LogBucket on the dev environment so I left this commented out. LMK if this should be updated.

Reply:
I'll try to provision a `cdo-dev-logs` bucket in the AWS Dev account to provide some parity.

Reply:
Hoping to complete setup of a log bucket in the dev account this morning.